devicon.json

```
@@ -3511,35 +3541,33 @@
           }
         ]
     },
-    {
-        "name": "spring",
+	{
+        "name": "subversion",
         "tags": [
-            "framework"
+            "svn",
+            "version"
         ],
         "versions": {
             "svg": [
                 "original",
-                "original-wordmark",
-                "plain",
-                "plain-wordmark"
+                "original-wordmark"
             ],
             "font": [
-                "plain",
-                "plain-wordmark"
+                "original",
+                "original-wordmark"
             ]
         },
-        "color": "#5FB832",
-        "aliases": []
-    },
-    {
-        "name": "spss",
-        "tags": ["package", "statistics"],
-        "versions": {
-          "svg": ["original", "plain"],
-          "font": ["plain"]
-        },
-        "color": "#cc1e4c",
-        "aliases": []
+        "color": "#809cc8",
+        "aliases": [
+            {
+                "base": "original",
+                "alias": "plain"
+            },
+            {
+                "base": "original-wordmark",
+                "alias": "original-wordmark"
```

Just need to change this to `plain-wordmark`. In the future, you don't have to worry about `aliases`. We are planning to phase it out eventually.
```suggestion
                "alias": "plain-wordmark"
```